Cargo.toml

```diff
@@ -100,10 +100,11 @@ gdal-sys = "0.10"
 # when changing geo version also adapt the Cargo.toml in the expression "deps-workspace"!
 geo = "0.29.3"
 geo-rand = { git = "https://github.com/lelongg/geo-rand", tag = "v0.3.0" }
+geo-types = "0.7.15"                                                       # important for compatibility when linking expressions
```

Comment:
as in the other comment: why do we need to specify it? it directly depends on the geo version or not?

Reply:
Siehe anderer Post.